src/providers/BackgroundBlurProvider/index.tsx

```diff
@@ -36,11 +36,14 @@ interface BackgroundBlurProviderState {
   isBackgroundBlurSupported: boolean | undefined;
 }
 
-const BackgroundBlurProviderContext =
-  createContext<BackgroundBlurProviderState | undefined>(undefined);
+const BackgroundBlurProviderContext = createContext<
```

Comment:
Shouldn't the 150 len make this one line?

Reply:
That is weird, do we need to change the prettier config as well and keep it in sync with the linter one?
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/main/.prettierrc#L2

Reply:
There was one more prettier config I had to fix. Done now.